config/settings.d/dns_dnscmd.yml.example

```diff
@@ -0,0 +1,8 @@
+---
+# Can be true, false
```

Comment:
Hah, I suppose, but in other places we also allow ```http``` and ```https``` values. I left the comment to keep it clear.